pkgs/development/python-modules/pg8000/default.nix

```
@@ -8,12 +8,12 @@
 
 buildPythonPackage rec {
   pname = "pg8000";
-  version = "1.19.0";
+  version = "1.19.2";
   disabled = pythonOlder "3.6";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "sha256-EexwwLIOpECAfiyGmUDxSE7qk9cbQ1gHtjhW3YK3RN0=";
+    sha256 = "sha256-RMu008kS8toWfKAr+YoAQPfpMmDk7xFMKNXWFSAS6gc=";
   };
 
   propagatedBuildInputs = [passlib scramp ];
```

Please update the substituteInPlace 

```suggestion
  propagatedBuildInputs = [ passlib scramp ];
```

---

Hmm, I thought that I fixed the `substituteInPlace` in 4327dfa. Apparently not, thanks for checking.